README.md

```diff
@@ -33,7 +33,8 @@ i.e., a Travis CI script. Here is a synopsis of a direct `tuttest` call:
     if you do not name a snippet, it will receive the `unnamedX` name, where
     `X` is the snippet number in the file (starting from 0). Examples of named
     snippets can be found below.
-  * `<prefix>` is a command that is added to the snippet as a prefix
+  * `<prefix>` is a command that is added to the snippet as a prefix.
+  * `--single-command` if provided, all snippets will be executed as a single command, separated with `;`.
```

Comment:
I would add in `<snippet_name>` that you can have multiple snippets executed if they are separated by a comma, and here, enhance the comment saying that all the snippets from `<snippet_name>` (if specified), will be executed all in a single command.